verification/curator-service/ui/package.json

```diff
@@ -64,7 +64,9 @@
     "test-silent": "react-scripts test --watchAll=false",
     "eject": "react-scripts eject",
     "cypress-open": "cypress open",
-    "cypress-run": "cypress run"
+    "cypress-run": "cypress run",
```

Comment:
Does plain `cypress run` use chrome or Electron? If it uses Electron can you add a command for chrome?

Reply:
whatever your default setting is locally, if you have chrome it will use chrome otherwise it will use electron I believe which is bundled with it.